README.md

```diff
@@ -25,12 +25,106 @@
+
+Let's change the background color of the main view to something other than white (because white is kind of boring, don't you think?). First, select the view. You can select the view by clicking anywhere in the blank white rectangle on your canvas. But _don't_ click on the label or button you created, or you will select them instead!
+
+Once the view is selected, bring up the Attributes Inspector. The Attributes Inspector is the _fourth_ button from the left in the right-hand sidebar. It kind of looks like a downwards-facing arrow:
+
+![Attributes Inspector button](.images/xcode07.png)
+
+Selecting that button will bring up the Attributes Inspector in Interface Builder's right-hand sidebar. Here you will see a number of options you can set:
+
+![Attributes Inspector](.images/xcode08.png)
+
+These attributes let you customize the look and feel of your views. You won't have to worry about most of them for now. For this lesson, the important one is the one labeled **Background**. As its name suggests, that attribute controls the color of the view's background. Go ahead and click on it to change the color to your favorite color (unless your favorite color is white, in which case change it to your second favorite color). Blue is nice, isn't it? After changing the color, your view will look something like this:
+
+![Changing the background color](.images/xcode09.png)
+
+### Initial View Controller
+
+One more thing: Right now, if you try to run your app, you'll get a warning that you haven't sent an initial view controller. The _initial view controller_ is the view controller that determines what the user sees when they first launch your application. Every application has to have an initial view controller. It's easy to designate a particular view controller as the _initial view controller_. First, make sure your view controller is selected in the object listing. You'll have to select it there, as you can't select it directly in the canvas:
+
+![Select the view controller](.images/xcode10.png)
+
+Once you've selected the view controller in the object listing, bring up its Attribute Inspector. Do you see an option marked **Is Initial View Controller** in the second section in the Attribute Inspector? Check that box to mark your view controller as the initial view controller.
+
+![Is Initial View Controller](.images/xcode11.png)
+
+Perfect! Now your view controller is in the initial view controller. You'll see that there is a grey arrow pointing into it to let you know that it is the starting point of your application.
+
+![Grey arrow](.images/xcode12.png)
+
+And that's it for this lesson! You can build and run your application to see it in the simulator. If you want, try adding more view components to your graphical interface and see what happens! You can also try changing some of the attributes for the label and button, and see how that affects them, too.
 
```

Comment:
I wonder how useful this would be for them to hear at this stage, but I'm thinking about adding something like this:

It is a good habit to get into to run your app and run it often, especially when you're making even the slightest change so that way as your views become more complex you can catch something very early as opposed to having to unravel many things you have done up until the last time you ran your application. Made adjustments to your label (perhaps changed the text or the background color), run the app before moving on to make adjustments to the button! It's a good habit to form which will help you when your views become more complex.

^ I wrote a lot there.. but feel free to nitpick it apart or make it much shorter.

@mdippery Thoughts?